UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/Reactions.cs

```diff
@@ -5,7 +5,7 @@ namespace Atmospherics
 	public static class Reactions
 	{
 		public const float T0C = 273.15f;
-		public const float PLASMA_MINIMUM_BURN_TEMPERATURE = T0C + 100;
+		public const float PLASMA_MINIMUM_BURN_TEMPERATURE = 200;
```

Comment:
This feels like a bit of a hacky fix since 200K is a very low temp for a fire.

The old value is from the original atmospherics as well so this would change the behaviour of plasma quite a bit:
![image](https://user-images.githubusercontent.com/4493127/71598738-06c9a100-2b40-11ea-849c-e090dce419af.png)

Reply:
I agree, I think the ticket is more about figuring out why the fires don't seem to spread anymore and if that's actually a problem. Apparently Bod is already able to cause large fires somehow anyway without this fix?

Reply:
yeah, you have to be persistent with the hot Spot until the room gets hot enough to cause it, I think one of the issues is that the game is so fast that it can't have time to build up heat inside of the area it's on fire so It goes out, Whilst in the byond version it's very slow and Has plenty of time to heat up the tiles the fire is on

Reply:
It seems to work roughly the same as byond ss13 with the 200k Setting,
apart from the massive fire explosion but I think that was there before The issue

Reply:
then maybe the issue is heat dissipating too quickly?

Reply:
heck, maybe it's not even an issue? I don't have a good sense of how easy it is to cause fires in us vs tg and whether it even should be changed.

Reply:
TG is Easy py, Only time I haven't seen fires spread with a Welding torch when I think there was too much plasma and too little oxygen And the torch was being dragged around as well, 99%  of time One Spark will set a massive fire off It's kind of that balance, like detective walks in with lit cigarette, One other issue is that Currently everything gets broken way to quickly From fire, like on TG doors never break to fire and Glasse as well, but Glass only breaks when there is a high pressure difference, and I find everything Burns up too quickly Something a super hot fire should last about 30 Seconds That's for items and  a lot of stuff doesn't even break if it's a gentle fire like Machines but this can be addressed later on

Reply:
*never breaks, I think doors only break when there's fire inside of itself

Reply:
okay, I think this issue should focus on fixing fires so they don't go out so quickly, without changing the temperature constants.

Reply:
if so, I should wait until foma has finished there atmospheric stuff since it might fix it

Reply:
I think it's fine to work on now, the atmos stuff foma is working on is NaN infection, which I don't think happens consistently with fires on OutpostStation, and shuttle crashing which also is an occasional thing, so this fix would already be on the happy path which should be protected from those bugs.

Reply:
If it is Strange behaviour, Since I remember it working before It might be fixed in one of those bugs